Rakefile

```diff
@@ -54,3 +54,5 @@ end
 
 require 'bundler/audit/task'
 Bundler::Audit::Task.new
+
+task default: ['test:all', 'rubocop', 'bundle:audit']
```

Comment:
This code is absolutely fine, but quick tip: I tend to write this using the [array of symbols shorthand](https://ruby-doc.org/core-2.3.1/doc/syntax/literals_rdoc.html#label-Percent+Strings):

```ruby
task default: %i(test:all rubocop bundle:audit)
```

✨